ethereum/contracts/ERC20App.sol

```diff
@@ -56,9 +57,9 @@ contract ERC20App {
         return abi.encode(_sender, _recipient, _tokenAddr, _amount, _nonce);
     }
 
-    function submit(bytes memory _data, bytes memory _signature)
+    function submit(bytes memory _data)
```

Comment:
In line with my previous comment above:
```suggestion
    function handle(bytes memory _message)
```

Reply:
https://github.com/Snowfork/polkadot-ethereum/pull/75/commits/31a105e146a248e1e665be92dab4405052e072b7